### PR TITLE
Optimize mobile scrolling experience by decreasing amount of information shown at once.

### PR DESCRIPTION
**Pull request by reschan**
Mobile-specific:
 - Add tabs/btns that switches between Equipment input and Ability Tree/Boost input.

All:
 - Condensed ID List information into two categories:
    + Summary: Shows the final total of HP/EHP, HPR, Eledefs and Any IDs that can't  be factored into damage calculation, spell cost calculation.
    + Detailed: Shows all IDs.
 - Adjusted some miscellaneous spacing issues and re-align the Equipment input to be in line with Damage and ID List.
 image

### CODE REVIEW

File: builder/index_full.html [hunk 1]
```diff
@@ -1118,28 +1128,271 @@
...
+                                <div class="row row-cols-1 row-cols-1 text-center scaled-font dark-5 rounded dark-shadow">
+                                    <div class="col fw-bold dark-4 rounded-top">
+                                        Active boosts
+                                    </div>
+                                    <div class="col mb-1">
+                                        <div class="col">
+                                            <button class="button-boost m-1 border-0 text-white dark-8u dark-shadow-sm" id="vanish-boost" onclick="update_boosts('vanish-boost')">
+                                                Vanish (+80%)
+                                            </button>
+                                            <button class="button-boost m-1 border-0 text-white dark-8u dark-shadow-sm" id="warscream-boost" onclick="update_boosts('warscream-boost')">
```
Comment (hppeng-wynn) [inline builder/index_full.html:1141]: Looks like duplicated code? Can you remove the old set

File: js/builder.js [hunk 2]
```diff
@@ -142,15 +142,18 @@ function toggle_boost_tab(tab) {
     document.getElementById(tab + "-boost-tab").classList.add("selected-btn");
 }
 
-let tabs = ['overall-stats', 'offensive-stats', 'defensive-stats'];
-function show_tab(tab) {
+// let tabs = ['overall-stats', 'offensive-stats', 'defensive-stats'];
+
+function show_tab(target, tabs) {
+    //console.log(itemFilters)
+
```
Comment (hppeng-wynn) [inline js/builder.js:149]: Is this the only one rn? can `toggle_boost_tab` above also be merged into this?

also can you document a bit exactly what the behavior is? it seems to depend on the ids' formatting being a specific way

**Review by hppeng-wynn — CHANGES_REQUESTED** [review]
some cleanup requested

File: builder/index_full.html (hunk 1 above)
Comment (reschan) [reply builder/index_full.html]: cleaned up duplicate codes

File: js/builder.js (hunk 2 above)
Comment (reschan) [reply js/builder.js]: done

**Review by hppeng-wynn — APPROVED** [review]

**Pull request merged**